CHANGES.rst

```diff
@@ -1,6 +1,14 @@
 Changes
 =======
 
+TBD
+---
+
+* Introduce ``ZyteAPIResponse`` and ``ZyteAPITextResponse`` which are subclasses
+  of ``scrapy.http.Response`` and ``scrapy.http.TextResponse`` respectively.
+  These new response classes hold the raw Zyte Data API response in its
+  ``zyte_api`` attribute.
```

Comment:
```suggestion
  These new response classes hold the raw Zyte Data API response in the
  ``zyte_api`` attribute.
```

Reply:
Or maybe “their”.

Reply:
the works for me

Reply:
Updated in https://github.com/scrapy-plugins/scrapy-zyte-api/pull/10/commits/37a4cc705d786a8d091b3066e6426b1f2dc9814c